### PR TITLE
IZE-113 infra lock

### PR DESCRIPTION
**Pull request by psihachina**
## Changelog:
- add dynamodb package
- add cli app
- update README

## Test:
[asciicast](https://asciinema.org/a/mgWnWYzm6mygx37idxJwYGdhE)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-# tflock
\ No newline at end of file
+# tflock
+Lock your terraform state manually.
+
+Until support only S3 backend.
```
Comment (DimmKirr) [inline README.md]:
```suggestion
Only S3 backend is supported for now.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+
+## Install
```
Comment (DimmKirr) [inline README.md]:
```suggestion
## Building
```

File: main.go [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+		Name:  "tflock",
+		Usage: "lock terraform state",
+		UsageText: `
+		# Lock terraform state in S3 (DynamoBD)
+		tflock --lock-id nutcorp-tf-state/env/terraform.tfstate
+		`,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "lock-id",
+				Usage:       "lock id",
```
Comment (DimmKirr) [inline main.go]:
```suggestion
				Usage:       "specify lock id, usually just a path to the state file in your s3 bucket",
```

File: main.go [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+				Required:    true,
+				Destination: &lockID,
+			},
+			&cli.StringFlag{
+				Name:        "aws region",
```
Comment (DimmKirr) [inline main.go]:
```suggestion
				Name:        "aws-region",
```

**Review by kobrikx — APPROVED** [review]
LGTM

File: main.go [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+				Value:       "us-east-1",
+			},
+			&cli.StringFlag{
+				Name:        "aws-profile",
+				Usage:       "AWS profile",
+				Required:    true,
+				EnvVars:     []string{"TFLOCK_AWS_PROFILE", "AWS_PROFILE"},
+				Destination: &profile,
+				Value:       "default",
+			},
```
Comment (DimmKirr) [inline main.go:50]: Let's add `dynamodb-table` parameter please

File: dynamodb/dynamodb.go [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+	sess, err := session.NewSessionWithOptions(session.Options{
+		Config:  *config,
+		Profile: profile,
+	})
+
+	if err != nil {
+		panic(err)
+	}
+
+	tableName := "tf-state-lock"
```
Comment (DimmKirr) [inline dynamodb/dynamodb.go:37]: Let's parametrize this please.

**Review by DimmKirr — CHANGES_REQUESTED** [review]

**Review by igorkotof — APPROVED** [review]

**Review by DimmKirr — COMMENTED** [review]
Looks good!

**Review by DimmKirr — APPROVED** [review]
Looks good!

**Pull request merged**